tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -622,8 +890,405 @@ public struct Monster: FlatBufferObject {
 		}
 		return nil
 	}
+
+
+	public mutating func unpack() -> MonsterT {
+		return MonsterT(&self)
+	}
+	public static func pack(_ builder: inout FlatBufferBuilder, obj: inout MonsterT?) -> Offset<UOffset> {
+		guard let obj = obj else { return Offset<UOffset>() }
+
+		let __name: Offset<String>
+		if let s = obj.name {
+			__name = builder.create(string: s)
+		} else {
+			__name = Offset<String>()
+		}
+
+		let __inventory = builder.createVector(obj.inventory)
+		let __test: Offset<UOffset>
+		switch obj.test?.type {
```

Comment:
You could consider putting this code in a function, since it will be repeated for each union field.